parameter/include/ParameterMgrFullConnector.h

```diff
@@ -71,10 +70,38 @@ class CParameterMgrFullConnector
     // Dynamic parameter handling
     CParameterHandle* createParameterHandle(const std::string& strPath, std::string& strError);
 
-    ISelectionCriterionTypeInterface* createSelectionCriterionType(bool bIsInclusive);
-    ISelectionCriterionInterface* createSelectionCriterion(const std::string& strName,
-            const ISelectionCriterionTypeInterface* pSelectionCriterionType);
-    ISelectionCriterionInterface* getSelectionCriterion(const std::string& strName);
+    /** Create a new Exclusive criterion
+     * Beware returned objects shall not be deleted by client.
+     * Should be called before start
+     *
+     * @param[in] name, the criterion name
+     * @param[in] values available value pairs for the created criterion
+     * @param[out] error the string describing the error if an error occurred
+     *                   undefined otherwise
+     * @return raw pointer on the criterion interface, nullptr if the method fail
+     */
+    core::selection::criterion::CriterionInterface*
+    createExclusiveCriterion(const std::string& name,
+                             const core::selection::criterion::CriterionInterface::Values& values,
```

Comment:
I'd prefer making it as easy+obvious for the user by simply taking a std::map<string, int> as argument.
You can make it even easier by overloading this method and take a std::list<string> (for people who don't care about the criterion numerical value).